plugins/inputs/internet_speed/internet_speed.go

```diff
@@ -129,11 +129,11 @@ func (is *InternetSpeed) findClosestServer() error {
 		client.SetNThread(is.Connections)
 	}
 
-	user, err := client.FetchUserInfo()
+	_, err := client.FetchUserInfo()
 	if err != nil {
 		return fmt.Errorf("fetching user info failed: %w", err)
 	}
```

Comment:
```suggestion
	
```

Reply:
I think, we should remove this step(FetchUserInfo) to save time.

Reply:
Looks like this info is used to determine the user's location. We do not use this diretly in any metrics. Based on the code it looks like this means we would then use Ookla's distance calc?

Reply:
Yes, we didn't use distance either. so it can be removed.

Reply:
thanks!

Reply:
Try with
```toml
[[inputs.internet_speed]]
```